Reserved field to make structure to be 256 bytes long, VESA BIOS
Extension 3.0 Specification says to reserve 189 bytes here but
that doesn't make structure to be 256 bytes.  So additional one is
added here.

diff -Nur vesautils-20050918.orig/libvbe/vbe.h vesautils-20050918/libvbe/vbe.h
--- vesautils-20050918.orig/libvbe/vbe.h	Fri Aug 12 20:49:53 2005
+++ vesautils-20050918/libvbe/vbe.h	Fri Jun 16 20:23:53 2006
@@ -134,7 +134,7 @@
   unsigned char  LinRsvdMaskSize;      /* 42: size of direct color reserved mask (linear modes) */
   unsigned char  LinRsvdFieldPosition; /* 43: bit position of lsb of reserved mask (linear modes) */
   unsigned long  MaxPixelClock;        /* 44: maximum pixel clock (in Hz) for graphics mode */
-  char           res4[189];            /* 48: remainder of ModeInfoBlock */
+  char           res4[189 + 1];        /* 48: remainder of ModeInfoBlock */
 }__attribute__ ((packed));
 
 typedef enum {
